jujugui/static/gui/src/app/components/unit-list/unit-list.js

```diff
@@ -52,6 +62,8 @@ YUI.add('unit-list', function() {
         groups[key].units.forEach((unit) => {
           this.refs['UnitListItem-' + unit.id].setState({
             checked: checked
+          }, () => {
+            this._updateActiveCount();
```

Comment:
I spent way to much time looking at this trying to figure out what was going on :) 

Maybe a quick comment here as to why it's making this call.